plugins/blockvolume/blockprovider/gluster-loopback/glusterloopback.go

```diff
@@ -0,0 +1,258 @@
+package glusterloopback
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/pkg/utils"
+	"github.com/gluster/glusterd2/plugins/blockvolume/blockprovider"
+	blkUtils "github.com/gluster/glusterd2/plugins/blockvolume/utils"
+
+	log "github.com/sirupsen/logrus"
+	config "github.com/spf13/viper"
+	"k8s.io/kubernetes/pkg/util/mount"
```

Comment:
@poornimag  do we have `Gopkg.toml ` changes for this one?

Reply:
if you are updating the dependencies using `dep ensure`, Please make sure version of dep to be `v0.5.0`